changelog_v0.1.md

```diff
@@ -499,11 +499,11 @@ List.nil |> List.cons 3.0 |> List.cons 2.0 |> List.cons 1.0 |> dyn |> map (inl x
 
 There is some convenience to doing the helpers directly as such. 
 
-Spiral can do runtime union types, but unlike with the MLs, they are not the main purpose of the language. Pattern matching on arbitrary types with no cost at compile time is. This also comes at the cost of having to write custom type inference code such as `type (Type meta: l) |> dyn |> f`.
+Spiral can do runtime union types, but unlike with the MLs, they are not the main purpose of the language. Pattern matching on arbitrary types with no cost at run time is. This also comes at the cost of having to write custom type inference code such as `type (Type meta: l) |> dyn |> f`.
```

Comment:
I see that this part has two interpretations. How about `Pattern matching on arbitrary types with no run time cost is.`